network-formula/network/wicked/common.sls

```diff
@@ -0,0 +1,41 @@
+{#-
+Salt state file for managing utilities for the Wicked Salt states
+Copyright (C) 2023 Georg Pfuetzenreuter <mail+opensuse@georg-pfuetzenreuter.net>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+-#}
+
+{%- from 'network/wicked/map.jinja' import base, base_backup, script -%}
```

Comment:
`base` is not used in this file, so either fix that or stop importing it

Reply:
ok